config/.jvm/src/main/scala/bloop/config/TargetPlatform.scala

```diff
@@ -0,0 +1,28 @@
+package bloop.config
+
+import java.nio.file.{Files, Paths}
+
+object TargetPlatform {
```

Comment:
What do you think of calling this `PlatformFiles` instead? Looks clearer at call-sites

Reply:
I'm all for it, fixed in aafbcae8c5d856f7045adc11114613e4c01ad0d6